src/constants.js

```diff
@@ -2,4 +2,3 @@ export const DOMAIN = 'accessible-colors.com';
 export const SITE_URL = `http://${DOMAIN}`;
 export const REPO = 'moroshko/accessible-colors';
 export const MIN_FONT_SIZE_IN_PX = 8;
-export const MAX_FONT_SIZE_IN_PX = 36;
```

Comment:
Can we leave the max for the input to be bounded? Why did we increase it to 36px?